committeeoversightapp/templates/committeeoversightapp/landing_page.html

```diff
@@ -1,10 +1,82 @@
 {% extends "base.html" %}
+{% load wagtailcore_tags %}
+{% load committeeoversight_tags %}
 
 {% block content %}
-  <div class="col-12 col-lg-9 mx-auto">
-    <h1 class="mb-4">{{page.title}}</h1>
+
+  <div class="row">
+    <div class="col-12 col-lg-4 col-xl-4 landing-description">
+        <h1 class="mb-4 ml-3 d-minus-rating">{{page.title}}</h1>
```

Comment:
this is a very minor detail, but for clarity's sake, would you mind defining a separate class for the header color? it could be `header-color`, if this is a common header color across pages, or something more generic like `orange-color`. i feel bad classifying our site title as d minus! 😂

Reply:
Ha very fair